mitodl/bootcamp-ecommerce
# Handle wire transfers

#### What are the relevant tickets?
Fixes #405 

#### What's this PR do?
Adds a management command to import wire transfers from a CSV (see Draft spreadsheet link in #405). 

#### How should this be manually tested?
 - I made an example CSV here: https://gist.github.com/noisecapella/9c093a86825b66f73df95afefad29bc6
 - Adjust the bootcamp run start date and bootcamp title to match the CSV, and your email to match the learner email (or adjust the CSV to match your existing data). 
 - Run `./manage.py import_wire_transfers ./example_wire_transfers.csv`
 - View the order history for the application. You should see the payment listed with payment type "Wire Transfer"
